devicon.json

```
@@ -3153,12 +3153,18 @@
         ],
         "versions": {
             "svg": [
+                "original",
+                "original-wordmark",
                 "plain",
-                "plain-wordmark"
+                "plain-wordmark",
+                "line",
+                "line-wordmark"
             ],
             "font": [
                 "plain",
-                "plain-wordmark"
+                "plain-wordmark",
+                "line",
+                "line-wordmark"
             ]
         },
         "color": "#b30000",
```

This is the color of the text in the lighter areas, and is also what you used in most of the icons' versions
```suggestion
        "color": "#bb0001",
```